solution/ui/prototype/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "vue-cli-service build",
     "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint",
-    "start-container": "docker compose up -d",
+    "start-container": "docker-compose up -d",
```

Comment:
do we need to change this?

Reply:
Doesn't work consistently across all systems whereas docker-compose still seems to